executor/index_merge_reader.go

```diff
@@ -717,3 +791,58 @@ func (w *indexMergeTableScanWorker) executeTask(ctx context.Context, task *looku
 	}
 	return nil
 }
+
+// IndexMergeRuntimeStat record the indexMerge runtime stat
+type IndexMergeRuntimeStat struct {
+	IndexMergeProcess time.Duration
```

Comment:
Should we record this information for each partial worker so that we can know which one is the bottleneck, which may be useful?

Reply:
If we want to know which partial worker is the bottleneck, maybe we can record the max time to find it?

Reply:
But there is only 1 `indexMergeProcessWorker` !!!

Reply:
No need to record each partition worker stats, since it already exists.

```sql
 IndexMerge_8                  | 3334.67 | 2       | root      |                        | time:1.209156ms, loops:2, index_task:{scan:223.583µs, merge:11.276µs}, table_task:{num:2, concurrency:5, time:3.533454ms}             
 ├─TableRangeScan_5(Build)     | 3333.33 | 0       | cop[tikv] | table:t2               | time:257.067µs, loops:1, cop_task: {num: 1, max:172.939µs, proc_keys: 0, rpc_num: 1, rpc_time: 117.418µs, copr_cache_hit_ratio: 0.00} 
 ├─IndexRangeScan_6(Build)     | 3333.33 | 0       | cop[tikv] | table:t2, index:t2a(a) | time:224.996µs, loops:1, cop_task: {num: 1, max:245.798µs, proc_keys: 0, rpc_num: 1, rpc_time: 152.734µs, copr_cache_hit_ratio: 0.00} 
```

partitial worker of `TableRangeScan_5` and `IndexRangeScan_6` already has individual execution info.